src/App.js

```diff
@@ -1,9 +1,11 @@
 import { BrowserRouter, Route, Routes } from "react-router-dom";
 import LandingPage from "./pages/LandingPage/LandingPage";
+import LoginPage from "./pages/LoginPage/LoginPage";
 import "./App.scss"
 import React from "react";
 import StaffRegistration from "./pages/StaffRegistration/StaffRegistration";
 import YouthRegistration from "./pages/YouthRegistration/YouthRegistration";
+import UserContext from "./context/UserContext";
```

Comment:
I have never worked with this before - what's it doing, how should it behave? Can you send me documentation

Reply:
https://reactjs.org/docs/context.html#when-to-use-context

Let me know if there's a better/different way you'd like to approach this implementation

Reply:
From what I understand, contexts are useful for properties that need to be accessible across all levels of the application. This pattern seemed like a good fit for user login data, and I haven't implemented this sort of thing before so I'm unfamiliar with the typical solution if use of contexts is uncommon

Reply:
nah after reading that totally makes sense I'm cool with it just wasn't familiar.